core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala

```diff
@@ -106,7 +106,7 @@ private[spark] abstract class BasePythonRunner[IN, OUT](
   protected val envVars: java.util.Map[String, String] = funcs.head.funcs.head.envVars
   protected val pythonExec: String = funcs.head.funcs.head.pythonExec
   protected val pythonVer: String = funcs.head.funcs.head.pythonVer
-
```

Comment:
Shall we keep the empty line to split the variable definition and `logInfo`?

Reply:
Done